src/main/java/core/basesyntax/TreeSetCharacters.java

```diff
@@ -16,8 +25,26 @@
  * <p>Приклад 2: A _f*c a?F
  * Результат 2: acf</p>
  */
+
 public class TreeSetCharacters {
-    public String getUniqueCharacters(String fileName) {
-        return null;
+    private static final int OUTPUT_LIMIT = 5;
+
+    public String getUniqueCharacters(String fileName) throws FileNotFoundException {
+        Set<String> treeSet = new TreeSet<>();
+        String line = "";
+        try {
+            BufferedReader input = new BufferedReader(new FileReader(fileName));
+            while ((line = input.readLine()) != null) {
```

Comment:
if you need to validate each character probable there is another method for this?